.pylintrc

```diff
@@ -60,7 +66,70 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-#disable=
+disable=
+    # These are disabled by default:
```

Comment:
I'm okay with leaving them enabled, for some reason I had the wrong impression that empty list == default.

Reading the [docs](https://pylint.readthedocs.io/en/latest/reference_guide/features.html#messages-control-options) again, I don't know what made me think that way...

Will remove/update.